frontend/src/components/user-details/components/user-profile/components/user-profile-form/select.data.ts

```diff
@@ -0,0 +1,11 @@
+import { UserGender } from 'common/enums/enums';
+import { SelectorOptions } from 'common/types/types';
+
+const genderOptions: SelectorOptions[] = Object.values(UserGender).map(
```

Comment:
let's create a helper in the `getGenderOptions` in the `frontend/src/components/user-details/components/user-profile/components/user-profile-form/helpers/get-gender-options/get-gender-options.helper.ts`

and use it in the component